osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/services/DeviceManagementService.java

```diff
@@ -186,7 +186,8 @@ public Organisation findOrganisation(@Identification final String organisationId
     @Transactional(value = "transactionManager")
     public Page<Event> findEvents(@Identification final String organisationIdentification,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 6 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS00107)

Reply:
Have you looked at introducing a class "SearchEventsCriteria"? It would reduce the number of parameters. In case we need more criteria later on, it's also easy to extend.

Reply:
done